tests/p2p/bootstrap_test/test_restart_bootstrap.py

```diff
@@ -1,5 +1,52 @@
-# from tests.conftest import init_session
-from tests.test_bs import setup_bootstrap_in_namespace
+from pytest_testconfig import config as testconfig
+import time
+
+from tests.conftest import DeploymentInfo
+from tests import pod
+from tests.test_bs import setup_bootstrap_in_namespace, add_curl, setup_bootstrap
+from tests.misc import CoreV1ApiClient
+
+
+# reboot bootstrap
+# add a new stateful bootstrap
+# kill bootstrap
+# create a new bootstrap with the former key
+# validate state is the same as the old bootstrap node
+def test_reboot_bootstrap(init_session):
+    print("test_reboot_bootstrap")
+    # test params
+    sleep_time = 5
+    session_id = init_session
+    bootstrap_key = 'bootstrap'
```

Comment:
This is actually bootstrap_label. Don't confuse it with bootstrap key

Reply:
fixing it